practice-project-ui/src/Calculator.tsx

```diff
@@ -1,25 +1,59 @@
 import * as React from "react";
+import axios from 'axios';
+
+const baseURL = "http://localhost:8000";
 
 function App() {
 
   const [data, setData] = React.useState(null);
 
-  React.useEffect(() => {
-    fetch("/api")
-      .then((res) => res.json())
-      .then((data) => setData(data.message));
-  }, []);
+  enum Operations {
+    Add = "+",
+    Subtract = "-",
+    Multiply = "*",
+    Divide = "/"
+  };
+
+  async function calculate(event: React.FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+
+    try {
+      axios.post(`${baseURL}/calculate`, {
+        firstNumber: (event.target as HTMLFormElement).firstNumber.value,
```

Comment:
Ditto comment as the API change, just use `first`

Reply:
done, but why?